utils/findLinks.js

```diff
@@ -0,0 +1,64 @@
+
+const chalk = require('chalk')
+
+
+const findLinks = (datos, urlF) => {
+    let allthelink = /\[(.*)\]\S((http|https):\/\/[^\s\n)]+)(?=\))/gim;
+    let regularL = /(http|https):\/\/.*\)?/gi;
+    let textReg = /\[(.*)\]/gim
+    let objectLinks = [];
+    let newFile = process.argv.indexOf('--file');
+    let uri = process.argv[newFile + 1];
+    let joinArray = []
+    let linkAndtext;
+    let joinObject;
+    return new Promise((resolve) => {
+        let noStr = datos;
+        let str = noStr.toString();
+        let myArray;
+        while ((myArray = allthelink.exec(str)) !== null) {
+            joinArray.push(myArray[0]);
+        }
+        linkAndtext = joinArray;
+        var unique = linkAndtext.filter(onlyUnique)
```

Comment:
en ES6 se recomienda el uso de let o const, 
```suggestion
        let unique = linkAndtext.filter(onlyUnique)
```